editor.planx.uk/src/@planx/components/Checklist/Public/Public.tsx

```diff
@@ -111,13 +91,9 @@ const VisibleChecklist: React.FC<Props> = (props) => {
   const flatOptions = getFlatOptions({ options, groupedOptions });
 
   const changeCheckbox = (id: string) => (_checked: any) => {
-    let newCheckedIds;
-
-    if (formik.values.checked.includes(id)) {
-      newCheckedIds = formik.values.checked.filter((x) => x !== id);
-    } else {
-      newCheckedIds = [...formik.values.checked, id];
-    }
+    const newCheckedIds = formik.values.checked.includes(id)
+      ? formik.values.checked.filter((x) => x !== id)
+      : [...formik.values.checked, id];
```

Comment:
I have a plugin that recommends cleaner ways to write code, it especially loves to refactor if...else blocks!

Reply:
I think switching the `if...else` to a ternary does not make it less readable imo :) sorry I made it sound like I was just letting the plugin run wild on its own (I definitely ignore a lot of what it says). I mainly made this change to reduce the number of lines of code without making any harder to understand. 🤞